src/advanced_inv.cpp

```diff
@@ -2407,28 +2432,14 @@ void advanced_inv_area::set_container_position()
             off = g->u.grab_point;
             break;
         case AIM_SOUTHWEST:
```

Comment:
I guess you can remove all cases except `AIM_DRAGGED` as they are still checked in new `aim_vector` function.

Reply:
Good point! Fixed!